bazel/ts_project/strict_deps/index.mts

```diff
@@ -44,16 +77,27 @@ for (const fileExecPath of manifest.testFiles) {
       }
     }
 
-    if (moduleSpecifier.startsWith('node:') && allowedModuleNames.has('@types/node')) {
+    if (
+      // node: prefixed module specifiers indicate this comes as a node builtin
+      (moduleSpecifier.startsWith('node:') ||
+        // if the module specifier matches ne of the built-in modules, we assume that it comes from the built-ins.
+        builtinModules.includes(moduleSpecifier)) &&
+      (allowedModuleNames.has('node') || tsconfig.options.types?.includes('node'))
+    ) {
```

Comment:
Node.js provide an API to check if the module is built-in.

```ts
import { isBuiltin } from 'node:module';
isBuiltin('node:fs'); // true
isBuiltin('fs'); // true
isBuiltin('wss'); // false 
```
See: https://nodejs.org/api/module.html#moduleisbuiltinmodulename


```suggestion
    if (
      isBuiltin(moduleSpecifier) &&
      (allowedModuleNames.has('node') || tsconfig.options.types?.includes('node'))
    ) {
```